.github/workflows/actions.yml

```diff
@@ -88,11 +88,11 @@ jobs:
       - name: Install Mockgen
         run: go get github.com/golang/mock/mockgen@v1.6.0
       - name: Get dependencies
-        run: go mod download
+        run: go mod download ; sudo apt -y install dnf
       - name: Mod tidy
         run: go mod tidy
       - name: Build podmon Docker Image
-        run: cd cmd/podmon ; make clean build; docker build -t podmon .
+        run: cd cmd/podmon ; chmod +x ../../scripts/buildubimicro.sh ; buildah unshare ../../scripts/buildubimicro.sh "registry.access.redhat.com/ubi8-micro:8.8-3"; make clean build ; podman build -t docker.io/podmon . --build-arg BASEIMAGE="localhost/resiliency-ubimicro" ; podman save docker.io/library/podmon -o /tmp/podmon.tar ; docker load -i /tmp/podmon.tar
```

Comment:
When an upgrade is needed for the ubi micro image, does it have to be updated here and in the [Makefile ](https://github.com/dell/karavi-resiliency/pull/168/files#diff-0bd6cb8d6cdc3dc4dabb736025ca1490030c842b20ff69b2240cba88fee5fa1e )as well?

Reply:
yes, was having issues running buildah with the github actions, and the only workaround I found was calling the script in this manner.